notebooks/csp/solving_csp_with_search.ipynb

```diff
@@ -195,6 +195,7 @@
     "# We know the search space is a tree.\n",
     "search_csp.layout_method = \"tree\"\n",
     "search_csp.text_size = 15\n",
+    "search_csp.simplify_graph = True\n",
```

Comment:
perhaps we could call the option hide_text?

Reply:
done.